sss_cal.dat

```diff
@@ -0,0 +1,306 @@
+113	-0.106600 0.046400 -0.060400 -0.012700 0.005700 -0.999903 -0.186801 -0.982403 -0.003300 -0.981487 0.190546 0.016294 -0.000907
```

Comment:
this file and the `cal` file should both go in the `mnefun/` source directory, not the repo root directory. In fact we should probably make a `mnefun/data/` directory if we don't have it yet and put them there